content/guides/moodle.md

```diff
@@ -0,0 +1,140 @@
+
+### Configure `config.php`
+
+Duplicate `config-dist.php` and rename it `config.php`. Update the following variables as follows:
+
+```php {filename="config.php", linenos=table}
+?php  // Moodle configuration file
+
+unset($CFG);
+global $CFG;
+$CFG = new stdClass();
+
+$CFG->dbtype    = 'mysqli';
+$CFG->dblibrary = 'native';
+$CFG->dbhost    = getenv("MYSQL_ADDON_HOST");
+$CFG->dbname    = getenv("MYSQL_ADDON_DB");
+$CFG->dbuser    = getenv("MYSQL_ADDON_USER");
+$CFG->dbpass    = getenv("MYSQL_ADDON_PASSWORD");
+$CFG->prefix    = 'mdl_';
+$CFG->dboptions = array (
+  'dbpersist' => 0,
+  'dbport' => getenv("MYSQL_ADDON_PORT"),
+  'dbsocket' => '',
+  'dbcollation' => 'utf8mb4_0900_ai_ci',
+);
+
+$CFG->wwwroot   = getenv("URL");
+$CFG->dataroot  = getenv("APP_HOME") . '/moodledata';
+$CFG->admin     = getenv("ADMIN");
+
+$CFG->directorypermissions = 0777;
+
+$CFG->sslproxy = true;
+
+require_once(__DIR__ . '/lib/setup.php');
+
+// There is no php closing tag in this file,
+// it is intentional because it prevents trailing whitespace problems!
+```
+
+Commit changes.
+
+### Declare the PHP Application
+
+On Clever Cloud Console, click on **Create** > **An application** and choose a [PHP](/doc/applications/php) application with Git deployment. Add a [MySQL](/doc/addons/mysql) add-on during the process.
+
+### Set Up Environment Variables
+
+Add the following [environment variables](/doc/develop/env-variables) to tour PHP application:
+
+```shell
+CC_PHP_VERSION="8"
+MAX_INPUT_VARS="5000"
+URL="<your-url"
+```
+
+If you don't have an domain for your Moodle applicaiton yet, you'll be able to add a test domain provided by Clever Cloud in step 6.
+
+### Set Up `moodledata` Folder
+
+This step will allow you to store your files outside of your application and [is required by Moodle to run](https://docs.moodle.org/403/en/Site_backup). All uploaded files and appearence set ups will be stored in a [File System Bucket](/doc/addons/fs-bucket) away from the appmlication server, as recommended by Moodle.
+
+Create an **FS Bucket add-on** and link it to your PHP application. In your FS Bucket dashboard, find the path variable (it should look like this: `CC_FS_BUCKET=/some/empty/folder:bucket-<bucket_id>`)
+
+Add this variable to your **PHP application** and replace `/some/empty/folder` by `/moodledata`. Don't forget to **update changes**.
+
+### Set Up Domain
+
+Moodle needs an URL declared in variables to work properly. You can set it up in **Domains names**, from your PHP application menu. If you don't have a domain name yet, you can use a `cleverapp.io` subdomain provided by Clever Cloud for test purposes.
+
+Don't forget to update `URL="<your-url"` if you haven't yet.
+
+### Deploy
+
+Get the remote in your application menu > **Information** > ** Deployùment URL** and add it to Git with `git remote add clever <clever-remote-url>`. Then, push your code with `git push clever -u master`
```

Comment:
* "Deployùment" has its weird accent  
* And the asterisks does not render boldly because of a whitespace.

Reply:
oups, fixed.